mlflow/store/artifact/hdfs_artifact_repo.py

```diff
@@ -288,7 +296,7 @@ def archive_artifacts(hdfs_artifact_repository, dest_path, archive_name):
             list_artifacts=list_artifacts, dest_path=dest_path)
     _logger.info("Command to execute to archive artifacts: %s", cmd)
     subprocess.check_output(shlex.split(cmd))
-    _, _, path_in_hdfs = _resolve_connection_params(dest_path)
+    _, _, _, path_in_hdfs = _resolve_connection_params(dest_path)
     return "har://hdfs-{host}{path_in_hdfs}/{archive_name}".format(
```

Comment:
It is great to have injected the path. Therefore now, we can use the correct schema in har path

Could you also remove the limitation I added to have hdfs in the har archive path, please ?

It should be "har://{schema}-{host}{path_in_hdfs}/{archive_name}"
for example "har://viewfs-pa4/user/j.doe/myarchive.har" is a valid name

Reply:
Done